### ISSUE TITLE
Problem with puppeteer version on Apple M1 chip

### ISSUE THREAD

**Issue by ugljanin** [gruntjs/grunt-contrib-qunit#184]
When I try to install your package version 5.1.1. 

```
/Volumes/Podaci/xwp/puppetier  npm i grunt-contrib-qunit
npm ERR! code 1
npm ERR! path /Volumes/Podaci/xwp/puppetier/node_modules/grunt-contrib-qunit/node_modules/puppeteer
npm ERR! command failed
npm ERR! command sh -c node install.js
npm ERR! The chromium binary is not available for arm64: 
npm ERR! If you are on Ubuntu, you can install with: 
npm ERR! 
npm ERR!  apt-get install chromium-browser
```

When I downloaded it from git and changed puppeteer version to above 10, it worked.
I have a problem because some (https://github.com/WordPress/wordpress-develop) package is using grunt-contrib-qunit in dependencies and it prevents me from installing it. Please update the version as I mentioned so I could work with it.

**Comment by talldan** [comment]
I've also encountered this. 

I think the issue is that the puppeteer dependency in this project is a fairly old version. That also means it's using an old version of Chromium, as puppeteer explicitly bumps the Chromium version on a regular basis (e.g. https://github.com/puppeteer/puppeteer/pull/7907).

My guess is this old version of Chromium doesn't have an ARM build, while newer versions do (I can verify that I haven't had the problem with newer versions of puppeteer).

So I think the solution would be to update this project to a much more recent version of Puppeteer.

**Comment by gziolo** [comment]
When can we expect this change to be published to npm? It looks like `6.0.0` release is already prepared: https://github.com/gruntjs/grunt-contrib-qunit/commit/3d09d410a707058a8972d92d6a4325721ecd17f6/.

**Comment by Krinkle** [comment]
Ref https://github.com/gruntjs/grunt-contrib-qunit/pull/186.
